Content.Client/Xenoarchaeology/Ui/NodeScannerBoundUserInterface.cs

```diff
@@ -0,0 +1,34 @@
+using Content.Shared.Xenoarchaeology.Equipment.Components;
+
+namespace Content.Client.Xenoarchaeology.Ui;
+
+public sealed class NodeScannerBoundUserInterface(EntityUid owner, Enum uiKey) : BoundUserInterface(owner, uiKey)
+{
+    [ViewVariables]
+    private NodeScannerDisplay? _scannerDisplay;
+
+    /// <inheritdoc />
+    protected override void Open()
+    {
+        base.Open();
+
+        _scannerDisplay = new NodeScannerDisplay(owner);
+        _scannerDisplay.OpenCentered();
+        _scannerDisplay.OnClose += Close;
+    }
+
+    public void Update(Entity<NodeScannerComponent> ent)
```

Comment:
sadly, those update methods are really underwhelming... i mean, they are proxy for ui update, you cannot disclose A LOT of information about them because its not correct to write down stuff in one place about stuff that is happening in the other place! When it will be changed - no one will correct xml-doc. :sadge: 
But i agree that any public method should have doc.